spec/services/doc_auth/mock/result_response_spec.rb

```diff
@@ -254,22 +254,22 @@
           glare_threshold: 40,
         },
       )
-      described_class.new(input, config, true)
+      described_class.new(input, config)
     end
 
     let(:input) do
       <<~YAML
         doc_auth_result: Passed
-        liveness_result: Fail
       YAML
     end
 
     it 'returns a passed result' do
       expect(response.success?).to eq(false)
       expect(response.errors).to eq(
-        general: [DocAuth::Errors::SELFIE_FAILURE],
-        selfie: [DocAuth::Errors::FALLBACK_FIELD_LEVEL],
-        hints: false,
+        back: [DocAuth::Errors::FALLBACK_FIELD_LEVEL],
+        front: [DocAuth::Errors::FALLBACK_FIELD_LEVEL],
+        general: [DocAuth::Errors::GENERAL_ERROR_NO_LIVENESS],
```

Comment:
Are these the right errors to expect here?

Reply:
This test case appears... contradictory. It's claiming that it's testing a passed result, but then asserting that it's unsuccessful?

I wonder if the `input` YAML is not set up correctly and is being treated as a failure (e.g. missing properties). Could we try pulling in a [passed result like what we document](https://developers.login.gov/testing/#testing-identity-proofing) and see if it would turn these values to a successful result?

Reply:
Agreed. The author of the test seems not sure which result they wanted to test. Per @jmhooper I used example .yaml files [from our docs](https://developers.login.gov/testing/#data-testing) to split this test into two:
- `returns a passed result`
- `returns caution result`